public/app/features/plugins/sql/components/configuration/useMigrateDatabaseFields.ts

```diff
@@ -1,8 +1,9 @@
 import { useEffect } from 'react';
 
 import { DataSourcePluginOptionsEditorProps } from '@grafana/data';
-import { logDebug, config } from '@grafana/runtime';
+import { config } from '@grafana/runtime';
 
+import { pluginsLogger } from '../../../utils';
```

Comment:
I think this may cause issues for us when externalising the sql data sources, @gabor am I correct?

Reply:
It still using `@grafana/runtime` to call `logDebug` so should not cause problems, but let me know.

Reply:
@ifrost sorry, we need to stay inside `public/app/features/plugins/sql` , this import goes to `public/app/features/plugins/utils.ts`.

Reply:
I've created a separate logger here: 2fabfd1b504f122d62468d82bdf5b1eb2755cf45